src/store/metadata/stub/content/editions/versions/total-value-of-uk-exports-2022-08.json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions",
+    "@type": "hydra:Collection",
+    "title": "Total value of UK exports",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-23T10:14:20",
+            "title": "Total value of UK exports",
+            "summary": "Estimated total value of UK exports including trade in both goods and services (experimental statistics), UK, 2021.",
+            "description": "This dataset shows the estimated total value of UK exports including trade in both goods and services (experimental statistics), in the UK, for 2021.\n\n### Coverage \n\nUK \n\n#### Geography definition\n\nInternational Territorial Level (ITL) 1, 2 and 3,\nCity Regions,\nCountry (UK)\n\n### Source\n\n - UK: [Office for National Statistics](https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/bulletins/internationaltradeinuknationsregionsandcities/previousReleases) \n\n\n",
+            "download_url": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                        "name": "Period",
+                        "datatype": "string",
+                        "titles": "Period",
+                        "description": "The time period associated with the observation."
+                    },
+                    {
+                        "name": "AREACD",
+                        "datatype": "string",
+                        "titles": "Statistical Geography",
```

Comment:
All other datasets use ONS Geography Code instead of Statistical Geography.

Reply:
Okay and noted